features/money_manager.feature

```diff
@@ -29,4 +29,11 @@ Scenario: Signing in should overwrite old answers with new ones
   When  I am on the Money Manager tool
   And   I answer the questions differently
   And   I sign in
-  Then  I have the latest answers
\ No newline at end of file
+  Then  I have the latest answers
+
+Scenario: Passing a special parameter in the url should show all content
+  Given I am signed in
+  And   I am on the Money Manager tool
+  And   I complete the series of questions
+  And   I visit the show all url
```

Comment:
Hi @scott-ad-riley I suspect the `Show All` will be a button to be clicked right? In which case rather than following a URL in this manner why not have this step instead `And I click the "Show all" button to open all advice tabs`.

Reply:
There is no button - this is an admin-only feature (without having to implement an admin and login). The people who are admins/content creators already know about the param for the url. 

This PR and the implementation one in universal credit are for upgrading the existing feature of the `?show_all` in the url - they aren't actually implementing that feature.

Reply:
@scott-ad-riley okay I get what you mean